vendor/github.com/openshift/installer/data/data/bootstrap/files/etc/sysconfig/fluentbit

```diff
@@ -0,0 +1 @@
+FLUENTIMAGE=arosvc.azurecr.io/fluentbit:1.3.9-1
```

Comment:
this needs to be updatable without having to revendor the installer.  Can you get all uses of this string into pkg/util/version/const.go and pipe them through somehow (I guess in the config struct?)

Reply:
added new fields, added a comment in genevalogging.go